README.md

```diff
@@ -42,7 +42,7 @@ If you want to scrobble / sync from Netflix, this is the only Trakt.tv [plugin](
 ### Which streaming services are supported?
 
 - Amazon Prime (Scrobble only)
-- HBO Go (Scrobble only - tested only for Latin America)
+- HBO Go (Only tested for Latin America, can only sync the 30 last watched items)
```

Comment:
I think it would be more fair to say that only Latin America is currently supported. I mean the URL is hardcoded to hbogola is it not?

Reply:
True, I actually think HBO Go is only available for Latin America. On US, it's HBO Max, and I'm not sure about the rest of the world.